x/sequencer/keeper/bond_reductions.go

```diff
@@ -30,7 +30,7 @@ func (k Keeper) completeBondReduction(ctx sdk.Context, reduction types.BondReduc
 		return types.ErrUnknownSequencer
 	}
 
-	if seq.Tokens.IsAllLT(sdk.NewCoins(reduction.DecreaseBondAmount)) {
```

Comment:
u assume single coin reduction, so the previous statement was correct as well.
I think dan suggested to use `!reduction.DecreaseBondAmount.IsAllGTE`, to cover the case for multi coins